index.html

```diff
@@ -0,0 +1,197 @@
+                                <div class="project-category text-faded"><span>WRX STI</span></div>
+                                <div class="project-name"><span>Project Boost</span></div>
+                            </div>
+                        </div>
+                    </a></div>
+                <div class="col-sm-6 col-lg-4"><a class="portfolio-box" href="assets/img/fullsize/gallery_01_06.jpg"><img class="img-fluid" src="assets/img/fullsize/gallery_01_06.jpg" alt="WRX STI">
+                        <div class="portfolio-box-caption">
+                            <div class="portfolio-box-caption-content">
+                                <div class="project-category text-faded"><span>WRX STI</span></div>
+                                <div class="project-name"><span>Project Boost</span></div>
+                            </div>
+                        </div>
+                    </a></div>
+                <div class="col-sm-6 col-lg-4"><a class="portfolio-box" href="assets/img/fullsize/gallery_01_07.jpg"><img class="img-fluid" src="assets/img/fullsize/gallery_01_07.jpg" alt="WRX STI">
+                        <div class="portfolio-box-caption">
+                            <div class="portfolio-box-caption-content">
+                                <div class="project-category text-faded"><span>WRX STI</span></div>
+                                <div class="project-name"><span>Project Boost</span></div>
+                            </div>
+                        </div>
+                    </a></div>
+                <div class="col-sm-6 col-lg-4"><a class="portfolio-box" href="assets/img/fullsize/gallery_01_09.jpg"><img class="img-fluid" src="assets/img/fullsize/gallery_01_09.jpg" alt="WRX STI">
+                        <div class="portfolio-box-caption">
+                            <div class="portfolio-box-caption-content">
+                                <div class="project-category text-faded"><span>WRX STI</span></div>
+                                <div class="project-name"><span>Project Boost</span></div>
+                            </div>
+                        </div>
+                    </a></div>
+                <div class="col-sm-6 col-lg-4"><a class="portfolio-box" href="assets/img/fullsize/interior360_01_00.jpg"><img class="img-fluid" src="assets/img/fullsize/interior360_01_00.jpg" alt="WRX STI">
+                        <div class="portfolio-box-caption">
+                            <div class="portfolio-box-caption-content">
+                                <div class="project-category text-faded"><span>WRX STI</span></div>
+                                <div class="project-name"><span>Project Boost</span></div>
+                            </div>
+                        </div>
+                    </a></div>
+            </div>
+        </div>
+    </section>
+    <section class="text-subaru-blue bg-dark">
+            <div class="container">
+                <div class="row">
```

Comment:
- [ ] The section "Featured speakers" should be  is generated dynamically with JavaScript.  The list of speakers used to generate the HTML is assigned to a variable as an array or object. The list of speakers should contain a speaker object for each that holds their specific data:


```
`const speakerList = [
{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}, 
{
  name: "Julia Leda2", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}, 
]
```

- [ ] Then you should have  a function that iterates through the speakers list on window load and creates HTML elements with the data from each object. For example:
```
window.onload = () => {
  speakerList.map(speaker => {
    // code to build the new HTML element, it can use `createElement()` or just template strings(``).
    // it could attach the new element to a parent inside within the loop
    // or accumulate the generate elements in a variable and attach them all after it
  })
}
```

Reply:
Created the speaker section in js.
`const app = [{
  cardClass: 'card',
  image: 'assets/img/thumbnails/gallery_01_10_thumb.jpg',
  alt: "Alex's car",
  title: 'Alex',
  subtitle: '2018 WRX',
  text: 'Nullam id dolor id nibh ultricies vehicula ut id elit. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec id elit non mi porta gravida at eget metus.',
  cardClass1: 'card',
  image1: 'assets/img/thumbnails/gallery_02_02_thumb.jpg',
  alt1: "John's car",
  title1: 'John',
  subtitle1: '2018 WRX',
  text1: 'Nullam id dolor id nibh ultricies vehicula ut id elit. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec id elit non mi porta gravida at eget metus.',
},
{
  cardClass: 'card',
  image: 'assets/img/thumbnails/gallery_01_13_thumb.jpg',
  alt: "Patrick's car",
  title: 'Patrick',
  subtitle: '2017 WRX STI',
  text: 'Nullam id dolor id nibh ultricies vehicula ut id elit. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec id elit non mi porta gravida at eget metus.',
  cardClass1: 'card',
  image1: 'assets/img/thumbnails/gallery_01_10_thumb.jpg',
  alt1: "Alex's car",
  title1: 'Alex',
  subtitle1: '2018 WRX',
  text1: 'Nullam id dolor id nibh ultricies vehicula ut id elit. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec id elit non mi porta gravida at eget metus.',
},
{
  cardClass: 'card',
  image: 'assets/img/thumbnails/gallery_02_02_thumb.jpg',
  alt: "John's car",
  title: 'John',
  subtitle: '2019 WRX STI',
  text: 'Nullam id dolor id nibh ultricies vehicula ut id elit. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec id elit non mi porta gravida at eget metus.',
  cardClass1: 'card',
  image1: 'assets/img/thumbnails/gallery_01_13_thumb.jpg',
  alt1: "Patrick's car",
  title1: 'Patrick',
  subtitle1: '2018 WRX',
  text1: 'Nullam id dolor id nibh ultricies vehicula ut id elit. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec id elit non mi porta gravida at eget metus.',
}];

function createCard(cardObject) {
  const div = document.createElement('div');
  div.className = 'container';
  div.innerHTML = `<div class="row" id="card-rows">
      <div class="col-md-6">
          <div class="card">
              <div class="card-body">
                  <img id="card-img" src="${cardObject.image}" alt="${cardObject.alt}" />
                  <h4 class="card-title">${cardObject.title}</h4>
                  <h6 class="text-muted card-subtitle mb-2 text-primary">${cardObject.subtitle}</h6>
                  <p class="card-text">${cardObject.text}</p>
              </div>
          </div>
      </div>
      <div class="col-md-6">
          <div class="card">
              <div class="card-body">
              <img id="card-img" src="${cardObject.image1}" alt="${cardObject.alt1}" />
                  <h4 class="card-title">${cardObject.title1}</h4>
                  <h6 class="text-muted card-subtitle mb-2 text-primary">${cardObject.subtitle1}</h6>
                  <p class="card-text">${cardObject.text1}</p>
              </div>
          </div>
      </div>
  </div>`;
  return div;
}

app.forEach((e) => {
  const card = createCard(e);
  cardSection.appendChild(card);
});`